src/controller/python/chip/yaml/runner.py

```diff
@@ -317,13 +352,21 @@ def _attribute_write_action_factory(self, test_step, cluster: str):
         except ParsingError:
             return None
 
+    def _wait_for_commissionee_action_factory(self, test_step):
+        try:
+            return WaitForCommissioneeAction(test_step, self._context)
+        except ParsingError:
+            return None
```

Comment:
what kind of parsing error can occur here? Could we add a comment explaining? Why do we return 'none' instead of allowing the exception to propagate?

Reply:
In the future we will want to propagate all these errors. For now ParsingError are for issues I know currently exist. and UnexpectedParsingError.

Would you be okay with just a TODO comment saying that we need to allow ParsingError to propagate once we have reached functional parity with codegen version of yaml parser with chip-repl?

Reply:
Chatted with Andrei about this. Just a TODO comment will be done. Since there are a few other *_factory method PRs in the work to prevent unneeded rebasing all *_factory method will be taken care of in a follow up PR

Reply:
This one spot has been addressed, but this comment more broadly for all *_factory methods will be done in a follow up PR